app/controllers/idv/gpo_verify_controller.rb

```diff
@@ -19,6 +19,10 @@ def index
       end
     end
 
+    def pii
+      Pii::Cacher.new(current_user, user_session).fetch
```

Comment:
Observing that the result of this (and what we pass to `save_in_person_enrollment`) is the `Pii::Attributes` struct, not a hash like in the default flow. Unsure if it's an issue, but we could also `.to_h` it.

Reply:
As far as blockery-ness, I think it is probably alright, since the struct allows for indifferent access and should allow it to be used like the hash form of it (i.e. `pii['first_name']` works).